### PR TITLE
3194 Export Coverage Continued

### PR DESCRIPTION
**Pull request by keeysnc**
### Description
Tests for the following export files. (update to ExportedReportFieldTable.tsx which also increases coverage)

1. ExportedModalOverlayReportSection.tsx
2. ExportedReportFieldTable.tsx

Screenshot 2024-01-30 at 12 32 04 PM

### Related ticket(s)

CMDCT-[MDCT-3194](https://qmacbis.atlassian.net/browse/MDCT-3194]

---
### Important updates

---
### Author checklist

- [ ] I have performed a self-review of my code
- [ ] I have added [thorough](https://shorturl.at/aejkF) tests, if necessary
- [ ] I have updated relevant documentation, if necessary
---

_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_

### CODE REVIEW

File: services/ui-src/src/components/export/ExportedModalOverlayReportSection.test.tsx [hunk 1]
```diff
@@ -1,137 +1,93 @@
...
 );
 
-describe("ExportedModalOverlayReportSection", () => {
+describe("Test ExportedOverlayModalReportSection Component", () => {
+  beforeEach(() => {
+    render(testComponent);
+  });
+  test("Test ExportRETTable render", () => {
+    screen.debug();
+  });
```
Comment (gmrabian) [inline services/ui-src/src/components/export/ExportedModalOverlayReportSection.test.tsx:81]: I think there should be some kind of assertion here

File: services/ui-src/src/components/export/ExportedModalOverlayReportSection.test.tsx [hunk 2]
```diff
@@ -1,137 +1,93 @@
...
+
+  test("should render status icon", async () => {
+    render(testComponent);
+    (renderStatusIcon as jest.Mock).mockReturnValue(HTMLElement);
+    expect(renderStatusIcon(true)).toEqual(HTMLElement);
+  });
```
Comment (gmrabian) [inline services/ui-src/src/components/export/ExportedModalOverlayReportSection.test.tsx:87]: `render(testComponent)` runs in the beforeEach so I don't think you need to repeat it. 

this is one of those tests where it seems like the mock is doing too much 😅 
could you instead render the real thing and verify that the image is present, or the alt text is present or something?

File: services/ui-src/src/components/export/ExportedOverlayModalReportSection.test.tsx [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+  </RouterWrappedComponent>
+);
+
+describe("Test ExportedOverlayModalReportSection Component", () => {
+  beforeEach(() => {
+    render(testComponent);
+  });
+  test("Test ExportRETTable render", () => {
+    screen.debug();
+  });
```
Comment (gmrabian) [inline services/ui-src/src/components/export/ExportedOverlayModalReportSection.test.tsx:74]: same as above: doesn't really prove it renders, does it? meaning, the test would pass the same if this was just a console.log(), right?

File: services/ui-src/src/components/export/ExportedReportFieldTable.test.tsx [hunk 4]
```diff
@@ -163,30 +152,7 @@ describe("ExportedReportFieldRow", () => {
...
-        },
-      ],
-      [],
-      [],
-      [],
-      [],
-    ];
-
-    await expect(getSectionFormFields).toEqual(result);
+    render(generalInformationComponent);
```
Comment (gmrabian) [inline services/ui-src/src/components/export/ExportedReportFieldTable.test.tsx:155]: needs an assertion

**Review by gmrabian — CHANGES_REQUESTED** [review]
See comments

File: services/ui-src/src/components/export/ExportedOverlayModalReportSection.test.tsx (hunk 3 above)
Comment (keeysnc) [reply services/ui-src/src/components/export/ExportedOverlayModalReportSection.test.tsx:74]: Yea this wasn't supposed to be there lol

**Review by gmrabian — APPROVED** [review]
tests pass, looks good!

**Pull request merged**